app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
+    @total_hits = FoodsFacade.total_hits(params[:q])
+    @food_results = FoodsFacade.food_results(params[:q])
```

Comment:
Can you think of a way to provide both hits and food data with just *one* API call?